idact/detail/add_cluster_app/main.py

```diff
@@ -29,13 +29,15 @@
               type=int,
               help="The ssh port. Default: 22")
 @click.option('--auth',
-              default=AuthMethod.PUBLIC_KEY,
-              type=AuthMethod,
-              help="Authentication method. Default: AuthMethod.PUBLIC_KEY")
+              default='GENERATE_NEW_KEY',
+              type=str,
+              help="Authentication method. "
+              "Avilable values: GENERATE_NEW_KEY, PRIVATE_KEY, ASK_EVERYTIME. "
+              "Default: GENERATE_NEW_KEY")
 @click.option('--key',
-              default=KeyType.RSA,
-              type=KeyType,
-              help="Specified key type to be generated. "
+              default="~/.ssh",
+              type=str,
+              help="Previously generated private key "
```

Comment:
Change the doc for something like `Path to the previously generated private key`